### PR TITLE
Docs edits

### PR DESCRIPTION
**Pull request by navidcy**
This PR is supersedes #154 and attempts to merge on `main` rather than on `joss-paper`.

### CODE REVIEW

**Comment by navidcy** [comment]
last commit I did `git commit -am"`pip` -> pip"` and the commit message came out all that man page!!!! :))))))

File: README.md [hunk 1]
```diff
@@ -122,7 +122,6 @@ necessarily those used by the GFDL's [`MOM6_examples`](https://github.com/NOAA-G
 
 The [example notebooks](https://regional-mom6.readthedocs.io/en/latest/demos.html) walk you through how to use
 the package using two different sets of input datasets.
-Please ensure that you can get at least one of these working on your setup with your MOM6 executable before trying modify the example to suit your domain with your bathymethry, forcing, and boundary conditions.
-
-You can download the notebooks [from Github](https://github.com/COSIMA/regional-mom6/tree/ncc/installation/demos) or by clicking on the download <img width="22" alt="download" src="https://github.com/COSIMA/regional-mom6/assets/7112768/2c1ae149-c6a8-4395-ab09-2f77588008d9"> button, e.g., at the top-right of the [regional tasmania forced by ERA5 example](https://regional-mom6.readthedocs.io/en/latest/demo_notebooks/reanalysis-forced.html).
+Please ensure that you can get at least one of these working on your setup with your MOM6 executable before trying modify the example to suit your domain with your bathymetry, forcing, and boundary conditions.
```
Comment (aekiss) [inline README.md:127]:
```suggestion
Please ensure that you can get at least one of these working on your setup with your MOM6 executable before trying to modify the example to suit your domain with your bathymetry, forcing, and boundary conditions.
```

**Review by aekiss — APPROVED** [review]
LGTM, thanks for fixing my PR

File: docs/mom6-file-structure-primer.md [hunk 2]
```diff
@@ -1,44 +1,47 @@
...
-Here we describe the various directories and files that `regional-mom6` package produces alongside with
-userful insights that, hopefully, will help users deal with troubleshooting and more advanced customisations.
+Here we describe the various directories and files that the `regional-mom6` package produces, and provide
+useful insights that, hopefully, will help users deal with troubleshooting and more advanced customisations.
 
 ## `run` directory
 
 The directory, specified by the `mom_run_dir` path keyword argument in the `experiment` class, contains only text files that configure MOM6 and are used at model initialisation.
-You can see examples of these files in the `premade_run_directories`.
+You can see examples of these files in [`docs/premade_run_directories`](https://github.com/COSIMA/regional-mom6/tree/main/demos/premade_run_directories).
```
[the remark this answers is not in the document]
Comment (navidcy) [reply docs/mom6-file-structure-primer.md:10]: fix typo

**Pull request merged**